templates/slapd.conf.erb

```diff
@@ -76,7 +78,7 @@ access to *
 # Define a config database (cn=config)
 
 database config
-suffix "cn=config"
+#suffix "cn=config"
```

Comment:
done but the build failed on Travis
but I have nothing to do with it :
https://travis-ci.org/datacentred/datacentred-ldap/jobs/56714578

should contain Package[ldap-gem] with ensure => "present" (FAILED - 1)



2015-04-01 11:05 GMT+02:00 Rob Greenwood <notifications@github.com>:

> In templates/slapd.conf.erb
> <https://github.com/datacentred/datacentred-ldap/pull/53#discussion_r27553827>
> :
>
> > @@ -76,7 +78,7 @@ access to *
> >  # Define a config database (cn=config)
> >
> >  database config
> > -suffix "cn=config"
> > +#suffix "cn=config"
>
> @maxdelorme <https://github.com/maxdelorme> Can you remove the line
> completely rather than just comment it out? Will merge after that. Thanks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datacentred/datacentred-ldap/pull/53/files#r27553827>.
>



-- 
-------------------------------------------------------------------------------------------------------
Maxime Delorme
Directeur du développement logiciel
4, rue du Faubourg Montmartre, 75009 Paris

Fixe :    +33 1 79 74 16 96
Mobile : +33 6 61 52 00 46
---------------------------------------------------------------------------------------------------------

-- 
Site @ : www.tennaxia.com

<http://www.tennaxia.com?signature=logo>
____________________________

  
<http://www.tennaxia.com/actualites/actualites-tennaxia/?signature=newsletter>  
<https://www.linkedin.com/company/tennaxia?signature=linkedin>   
<http://www.viadeo.com/fr/company/tennaxia?signature=viadeo>   
<https://twitter.com/Tennaxia?signature=twitter>   

*Adoptez l'éco-attitude. N'imprimez ce mail que si cela est vraiment 
nécessaire !*